### DISCUSSION TITLE
Can no longer find the option to change to miles

### DISCUSSION THREAD

**Discussion by chris2172** [Hyundai-Kia-Connect/kia_uvo#502] [General]
I noticed that the setting had reverted back to km and went into intergrations and vlicked the configure button to change it back to miles and it is no longer there.
![image](https://user-images.githubusercontent.com/11930805/207987636-29e99271-6c5c-4a05-9188-e8b42803cdba.png)

**Comment by cdnninja** [comment]
If you go into the entity itself you will find it built into home assistant now.  It doesn't require us to create a custom feature for this. 
![image](https://user-images.githubusercontent.com/6373468/207987934-9ddd1d82-9760-45d7-b38f-31a1e34ef084.png)

**Comment by chris2172** [comment]
Thanks, I have changed in in Odometer and Fuel Driving Range

**Reply by cdnninja** [reply]
Great!

**Comment by festejester** [comment]
I see the drop down for the odometer, but no drop down in the three “range” items.

**Reply by cdnninja** [reply]
Which region and does it show as KM?  Which version are you running?

**Reply by festejester** [reply]
I am in US.  

Interestingly, in my car the odometer is in miles (6400) but the app says 10300 mi, but I think it really is km.  Strange

**Reply by cdnninja** [reply]
> I am in US.  
> 
> 
> 
> Interestingly, in my car the odometer is in miles (6400) but the app says 10300 mi, but I think it really is km.  Strange

Can you open an issue. I'll need to know the brand and the data sensor attributes.

**Comment by booti101** [comment]
@cdnninja 
I'm new to the app, so apologies if this has been asked and answered before.
I have managed to change all the distance settings to Miles, but i cant find a way to adjust the Average Energy Consumption entity from Wh/km to Wh/mile

**Reply by cdnninja** [reply]
We currently don't have a way to convert that one.

**Reply by booti101** [reply]
OK, thanks for confirming
